datastore/kafka/kafka.go

```diff
@@ -130,6 +132,18 @@ func (p *Producer) logError(err error) {
 	metricsRegistry.errorCount.Inc(map[string]string{})
 }
 
+func (p *Producer) reportProducerMetrics() {
+	interval := time.Minute
```

Comment:
One minutes feels like a really long interval; 5s? It should be super cheap right?

Reply:
updated